test/Dialect/XTenNN/ops_invalid.mlir

```diff
@@ -24,3 +24,32 @@ func.func @mish_int(%arg0: tensor<1x10xi4>) -> tensor<1x10xi4> {
     %0 = xten_nn.mish %arg0 : (tensor<1x10xi4>) -> tensor<1x10xi4>
     return %0 : tensor<1x10xi4>
 }
+
+// -----
+
+func.func @kernel_missing_parenthesis() {
+    // expected-error@+1 {{expected '('}}
+    %a = xten_nn.kernel "myKernel" -> tensor<2xi64>
+}
+
+// -----
+
+func.func @kernel_missing_type(%arg0: i8, %arg1: i8) {
+    // expected-error@+1 {{expected ':'}}
```

Comment:
Is there a way to specify nicer errors? I.e. it seems we could hint here something like "kernel argument format is val : type, can't find :"

Reply:
this turned out to be possible, so done